src/objects/charts/chart--horizontal-bar.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+import Vue from 'vue/dist/vue.esm.browser';
+import ChartComponent from './chart--horizontal-bar.vue'; // Our component
+import ChartData from 'nyco-patterns/src/objects/charts/chart.data'; // Our sample data
```

Comment:
can we update the path to './chart.data'?

Reply:
ah, yes! thanks for catching. I think this was leftover from my issues with building nyco-patterns inside /node_modules. will update.